src/Protos/Grpc/redaction.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package event_store.client.redaction;
+option java_package = "com.eventstore.dbclient.proto.redaction";
+
+import "shared.proto";
+
+service Redaction {
+	rpc GetEventPosition (stream GetEventPositionReq) returns (stream GetEventPositionResp);
+	rpc SwitchChunk (stream SwitchChunkReq) returns (stream SwitchChunkResp);
+}
+
+message GetEventPositionReq {
+	event_store.client.StreamIdentifier stream_identifier = 1;
+	uint64 stream_revision = 2;
+}
+
+message GetEventPositionResp {
+	repeated EventPosition event_positions = 1;
+}
+
+message EventPosition {
+	uint64 log_position = 1;
+	ChunkInfo chunk_info = 2;
+}
+
+message ChunkInfo {
```

Comment:
bumping a little on this being called 'ChunkInfo' but then having an event specific offset in it. would it be senseible to just encororate these fields directly into the eventposition?